R/s3functions.R

```diff
@@ -36,11 +36,20 @@ plot.diversityEstimates <- function(x, ...) {
   }
   xx <- args$xx
   h0 <- args$h0
-  
-  lci <- dv[[h0]] - 2*sqrt(dv[[paste(h0, "-variance", sep = "")]])
-  uci <- dv[[h0]] + 2*sqrt(dv[[paste(h0, "-variance", sep = "")]])
-  df <- data.frame("names" = names(dv[[h0]]), 
-                   "h0" = dv[[h0]], lci, uci, dv$X)
+  if (h0 %in% c("shannon", "simpson")) {
+    ests <- sapply(dv[[h0]], function(x) x$estimate)
+    vars <- sapply(dv[[h0]], function(x) x$error)
+    lci <- ests - 2*sqrt(vars)
```

Comment:
I think we should use the constructed intervals:
lci <- sapply(dv[[h0]], function(x) x$interval[1])
uci <-  sapply(dv[[h0]], function(x) x$interval[2]))

Reply:
d57f724357612b30ff56122843ff0129befb4cf3